keyboards/newgame40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,213 @@
+ ),
+
+ /* Lower
+  * ,---------------------------------------------------------------------.
+  * |  Esc |      |      |      |      |      |      |      |      |      |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |      |      |      |      |      |      |      |      |      |      |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |  F13 |  F14 |  F15 |  F16 |  F17 |  F18 | F19  | F20  | F21  | F22  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |  F1  | F2   |      | F3   |  F4  |  F5  |  F6  |      |  F7 |   F8  |
+  * `---------------------------------------------------------------------'
+  */
+ [_LOWER] = LAYOUT( \
+   KC_ESC,  _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+   KC_F13,  KC_F14,  KC_F15,  KC_F16,  KC_F17,  KC_F18,  KC_F19,  KC_F20,  KC_F21,  KC_F22,  \
+   KC_F1,   KC_F2,   _______, KC_F3,   KC_F4,   KC_F5,   KC_F6,   _______, KC_F7,   KC_F8    \
+ ),
+
+ /* Raise
+  * ,---------------------------------------------------------------------.
+  * |   1  |   2  |　 3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   !  |   "  |   #  |   $  |   %  |   &  |   '  |  (　 |   )  |   _  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   =  |   ¥  |  {   |   }  |   *  |   ~  |   |  |  <   |   >  |   ?  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |  F1  |  Alt |      |   [  |   ]  |   ;  |   :  |      |   \  |   /  |
+  * `---------------------------------------------------------------------'
+  */
+ [_RAISE] = LAYOUT( \
+   KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, \
+   KC_EXLM, KC_DQT,  KC_HASH, KC_DLR,  KC_PERC, KC_AMPR, KC_QUOT, KC_LPRN, KC_RPRN, KC_UNDS,  \
+   KC_EQL,  KC_JYEN, KC_LCBR, KC_RCBR, KC_ASTR, KC_TILD, KC_PIPE, KC_LABK, KC_RABK, KC_QUES,  \
+   KC_F1,   KC_LALT, _______, KC_LBRC, KC_RBRC, KC_SCLN, KC_COLN, _______, KC_BSLS, KC_SLSH   \
+ ),
+
+ /* EUCALYN
+  * ,---------------------------------------------------------------------.
+  * |   Q  |   W  |   ,  |   .  |   ;  |   M  |   R  |   D  |   Y  |   P  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   O  |   E  |   I  |   U  |   G  |   T  |   K  |   S  |   N  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   Z  |   X  |   C  |   V  |   F  |   B  |   H  |   J  |   L  |   /  |
+  * `-------------+------+------+------+------+------+------+-------------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_EUCALYN] = LAYOUT( \
```

Comment:
If you're using `LAYOUTS = ortho_4x10`, then you should use that here: 

```suggestion
 [_EUCALYN] = LAYOUT_ortho_4x10( \
```